data/thebarshablog/events/2023-02-21-Decoding-CFPs-Twitter-Space.json

```
@@ -0,0 +1,12 @@
+{
+  "isVirtual": true,
+  "isInPerson": false,
+  "color": "purple",
+  "name": "Decoding CFPs (Call for Proposals)",
+  "description": "Twitter Space",
+  "date": {
+    "start": "2023-02-21 T16:00:00.000+00:00",
+    "end": "2023-02-21 T17:00:00.000+00:00"
```

I think there should be no space between the date & time in this format. 🤔

```suggestion
    "start": "2023-02-21T16:00:00.000+00:00",
    "end": "2023-02-21T17:00:00.000+00:00"
```